### PR TITLE
Maryia/91623/fix: binary bot did not switch to the expected language

### PR DESCRIPTION
**Pull request by maryia-matskevich-deriv**
## Changes

-   fix: binary bot did not switch to the expected language, shouldn't switch to russian language 

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by kate-deriv — APPROVED** [review]

**Review by vinu-deriv — APPROVED** [review]

**Review by sandeep-deriv — APPROVED** [review]

**Comment by rupato-deriv** [comment]
Why did we remove russian language when the card says redirect to english ?

**Review by rupato-deriv — APPROVED** [review]

**Review by farabi-deriv — APPROVED** [review]

**Review by sandeep-deriv — APPROVED** [review]

**Review by prince-deriv — APPROVED** [review]

**Review by vinu-deriv — APPROVED** [review]

File: src/common/lang.js [hunk 1]
```diff
@@ -4,14 +4,14 @@ import { setCookieLanguage } from '../common/utils/cookieManager';
 import { supportedLanguages, translate, init } from './i18n';
 
 export const getLanguage = () => {
-    const queryLang = parseQueryString().l ? parseQueryString().l || getStorage('lang') : 'en';
-
+    const queryLang = parseQueryString().lang ? parseQueryString().lang || getStorage('lang') : 'en';
+    const isQueryLangSupported = queryLang in supportedLanguages;
```
Comment (prince-deriv) [inline src/common/lang.js:8]: Can we convert the variables to `snake_case`?

Comment (maryia-matskevich-deriv) [reply src/common/lang.js]: let me try it, the project setup didn't allow it before, but I may be confused with an older project binary bot

Comment (maryia-matskevich-deriv) [reply src/common/lang.js]: done ✅

**Review by prince-deriv — APPROVED** [review]

**Review by shafin-deriv — APPROVED** [review]

**Review by sandeep-deriv — APPROVED** [review]

**Pull request merged**